v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples_test.py

```diff
@@ -74,6 +74,12 @@
 }
 
 
+@patch(
```

Comment:
I guess another solution here... can we just mock `UpdateVariantAnnotationsTableWithUpdatedReferenceDataset` into the `MockCompleteTask`?  that might be a clean option?

Reply:
oh yes that's cleaner